website/views/shared/avatar/appearance.jade

```diff
@@ -1,6 +1,7 @@
 mixin avatar(opts)
   .character-sprites
     .addthis_native_toolbox(ng-if='::profile._id==user._id',
+      ng-mouseover='siHovered=true' ng-mouseout='siHovered=false' ng-init='siHovered=false'
```

Comment:
This shouldn't require `ng-init` since `isHovered` will start out `undefined` which is a falsy value.